src/test/java/uk/gov/digital/ho/hocs/queue/common/QueueListenerTest.java

```diff
@@ -35,15 +35,16 @@ public void messageTypeDoesNotMatch_doNothing() throws Exception {
         verifyNoMoreInteractions(ukviComplaintMessageHandler);
     }
 
+
     @Test
+    @DirtiesContext(methodMode = DirtiesContext.MethodMode.AFTER_METHOD)
     public void whenMessageShouldBeIgnored_doNothing() throws Exception {
+        ReflectionTestUtils.setField(queueListener, "shouldIgnoreMessages", true);
```

Comment:
I'm not sure about using reflection here but other than splitting the ignore message tests into a separate class I don't think there's a way around it?

Reply:
Yeah I weren't a fan of the reflection but I'm not a fan of having a specific test just for this either.